5.py

```diff
@@ -0,0 +1,70 @@
+#1
+n = int(input())
+list = list(map(int, input().split()))
+print(min(list), max(list))
+
+#2
+list = []
+for i in range(9): # 리스트 생성 후 append 함수로 input 추가하기
+    list.append(int(input()))
```

Comment:
파이썬 리스트 컴프리헨션 공부해보세요~! 훨씬 깔끔하죠?! 😀 
파이썬 프로그래머라면 컴프리헨션은 필수죠! ㅎㅎ
```suggestion
list = [int(input) for _ in range(9)]
```

Reply:
확실히 한 줄로 간결하게 돼 있어서 보기 편하네요. 굿.
파이썬 리스트 컴프리헨션에 대해 알아보겠습니다! 감사합니다.